src/main/kotlin/dev/binclub/paperbin/transformers/OptimisedEveryoneSleeping.kt

```diff
@@ -40,12 +44,12 @@ object OptimisedEveryoneSleeping: PaperFeature {
 	 * This also fixes MC-47080
 	 */
 	@JvmStatic
-	fun everyoneDeeplySleeping(worldServer: Any): Boolean {
+	fun everyoneDeeplySleeping(flag: Boolean, worldServer: Any): Boolean {
 		worldServer as WorldServer
 		
 		val players = worldServer.players
-		
-		return if (players.size == 0)
+
+		return if (players.size == 0 || worldServer.isClientSide || !flag)
```

Comment:
When is `isClientSide` true?
Isn't this a server?

Reply:
idk, mc code checks that